specutils/tests/test_loaders.py

```diff
@@ -1355,3 +1355,19 @@ def test_spectrum_list_directory_miri_mrs(tmpdir):
     for spec in specs:
         assert isinstance(spec, Spectrum1D)
         assert spec.spectral_axis.unit == u.micron
+
+
+# loading a Spectrum1D from a MIRI x1d file
+@remote_access([
+    {'id': "5363560", 'filename':"jw00623-c1012_t002_miri_p750l_x1d.fits"},     # pipeline 1.2.3
+    {'id': "5363560", 'filename':"jw00212-o035_t007_miri_ch2-medium_x1d.fits"}, # pipeline 1.1
+    {'id': "5363560", 'filename':"jw00623-o057_t008_miri_ch1-long_x1d.fits"},   # pipeline 1.3.1
+])
+def test_jwst_miri_x1d(remote_data_path):
+
+    data = Spectrum1D.read(remote_data_path)
+
+    assert type(data) is Spectrum1D
```

Comment:
Is there a reason to not use `isinstance`?

```suggestion
    assert isinstance(data, Spectrum1D)
```

Reply:
Not really. I copied the code pattern from a number of similar tests in `test_jwst_reader.py.` Then I moved the test to `test_loaders.py` so it could benefit from the remote loading fixture that is already in place there. `test_jwst_reader.py` uses only internally-generated test data. I will change that to `isinstance`